lib/house.rb

```diff
@@ -0,0 +1,61 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price.delete("$").to_i
```

Comment:
I think that I would keep the price as a string and have a method do the conversion so we don't lose information.